test/strategies/mcd/mcd-tests.js

```diff
@@ -133,45 +148,31 @@ const mcdBoostStrategyTest = async () => {
             await redeploy('McdPayback');
             await redeploy('McdOpen');
             await redeploy('McdRatio');
+            await redeploy('McdBoostComposite');
+            await redeploy('McdFLBoostComposite');
             strategyTriggerView = await redeploy('StrategyTriggerView');
             await addBotCaller(botAcc.address);
 
             proxy = await getProxy(senderAcc.address);
         });
 
-        it('... should create 2 boost strategies and create a bundle', async () => {
+        it('... should create 4 boost strategies and create a bundle', async () => {
             const boostStrategy = createMcdBoostStrategy();
```

Comment:
Switch normal recipe strategies, to flBalancer as well for better gas comparison